build/lin.sh

```diff
@@ -468,6 +470,13 @@ fi
   ${DARWIN:+--disable-Bsymbolic}
 make install-strip
 
+mkdir ${DEPS}/cgif
+$CURL https://github.com/dloebl/cgif/archive/V${VERSION_CGIF}.tar.gz | tar xzC ${DEPS}/cgif --strip-components=1
+cd ${DEPS}/cgif
+CFLAGS="${CFLAGS} -O3" meson setup _build --default-library=static --buildtype=release --strip --prefix=${TARGET} ${MESON}
```

Comment:
Maybe we should avoid compiling the unit tests?
```suggestion
CFLAGS="${CFLAGS} -O3" meson setup _build --default-library=static --buildtype=release --strip --prefix=${TARGET} ${MESON} \
  -Dtests=false
```